e2e/cases/assets-retry/index.test.ts

```diff
@@ -5,10 +5,23 @@ import type { PluginAssetsRetryOptions } from '@rsbuild/plugin-assets-retry';
 import { pluginReact } from '@rsbuild/plugin-react';
 import type { RequestHandler } from '@rsbuild/shared';
 
+const pattern = [
+  '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]+)*|[a-zA-Z\\d]+(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)',
+  '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-ntqry=><~]))',
+].join('|');
+
+const ansiRegex = new RegExp(pattern, 'g');
+
+function stripAnsi(content: string) {
```

Comment:
how about use the strip-ansi package

https://www.npmjs.com/package/strip-ansi

Reply:
done✅